shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -297,6 +297,9 @@ - (void)mouseDown:(NSEvent*)event {
   // See: https://github.com/flutter/flutter/issues/115015
   // See: http://www.openradar.me/FB12050037
   // See: https://developer.apple.com/documentation/appkit/nsresponder/1524634-mousedown
+  //
+  // TODO(cbracken): https://github.com/flutter/flutter/issues/154063
+  // Remove this workaround when we drop support for macOS 12 (Monterey).
```

Comment:
Instead of a todo you could instead wrap this in an `@availability` check, I usually audit and remove the out-of-date ones when we bump support because I love deleting code https://github.com/flutter/engine/pull/34919
```objc
if (@available(macOS 13.3.1)) {
  // The bug that required this workaround was fixed in macOS 13.3.1.
} else {
  [self.nextResponder mouseDown:event];
}
```

Reply:
![image](https://github.com/user-attachments/assets/ae993c31-7981-46dd-9450-69136f914ff7)

Good call -- will send a patch.

Reply:
Sent: https://github.com/flutter/engine/pull/54784